bruno/collections/Rafiki/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "rafiki-bruno-collection",
+  "version": "1.0.0",
+  "dependencies": {
+    "json-canonicalize": "^1.0.6"
+  }
+}
```

Comment:
I think we should probably add this as a package to our pnpm workspace so `pnpm i` will install these dependencies. Took me some time to figure out why bruno couldn't find `json-canonicalize`. 

In fact I'm not even sure how to install these with pnpm otherwise... doing `pnpm i` from `bruno/collections/Rafiki` just installs the workspace packages (and not this).

Reply:
I think I managed it by running `pnpm init` inside the directory to generate the `package.json` on my machine. But you're right, it should be listed in the pnpm workspace.